backend/src/database/entities/availability.entity.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  Column,
+  Entity,
+  PrimaryGeneratedColumn,
+  ManyToOne,
+  JoinColumn,
+} from 'typeorm';
+import { AvailabilityType } from '../../common/enums/availability-type.enum';
+import { PersonnelEntity } from './personnel.entity';
+
+@Entity('function')
+export class AvailabilityEntity {
+  @PrimaryGeneratedColumn('increment')
+  id: string;
+
+  @Column({ name: 'date', type: 'timestamp' })
+  date: Date;
```

Comment:
I feel like we will need a start date and an end date here

Reply:
I thought it easier to define it day by day rather than a date range, but I'm open to discussion. This can be finalized when we're doing more of the availability / scheduling iscussions

Reply:
I vote not even adding in the availability until we do the scheduler. But I am also fine to include an update later.